assets/pages/layout.twig

```diff
@@ -473,9 +474,9 @@
                       09.00-12.00 AM</p>
                       <p>Saturday and Sunday<br>
                       closed</p>
-                      <p><a href="#" class="icon icon--before icon--print">Print contact infos</a></p>
+                      <p><a href="#" onclick="$.printPreview.printPreview('contact')" class="icon icon--before icon--print">Print contact infos</a></p>
```

Comment:
This look a bit like black magic. What about this `onclick="$.printPreview.printPreview($('[data-print=contact]'))"` ?

More verbose but clearer I think.